src/app/ReadClient.cpp

```diff
@@ -519,13 +519,16 @@ void ReadClient::OnPeerTypeChange(PeerType aType)
 {
     VerifyOrDie(mpImEngine->InActiveReadClientList(this));
 
-    mIsPeerLIT = (aType == PeerType::kLITICD);
+    if (mReadPrepareParams.mIsPeerLIT)
```

Comment:
I have no idea how to review this, because mIsPeerLIT on ReadPrepareParams is completely undocumented.  What does that boolean even mean?  The PR that added it also does not explain what it's really meant to do and why....

Does this field mean "we have set up a check-in target on the peer" or something else?  (Especially given that SITs can support check-in.)

Reply:
Yes, I think it should be "we have set up a check-in target on the peer", I have renamed mIsPeerLIT to mRegisteredCheckInToken in ReadPrepareParams, considering most ecosystems are still developing stage, either way they need update this parameter diligently, I think it is fine to modify this directly instead of creating another parameter.